### ISSUE TITLE
Use "an" for 1.0, 1px, etc.

### ISSUE THREAD

**Issue by tesseralis** [retextjs/retext-indefinite-article#13]
### Subject of the issue

Similar to #5 #7  and #11, we get the wrong 

### Steps to reproduce

Run retext on something like:

```
Version 1.0.2 moves everything left by 1px.
```

### Expected behavior

Passes with no errors.

### Actual behavior

```
[...]  warning  Use `an` before `1px`, not `a`             retext-indefinite-article  retext-indefinite-article
[...]  warning  Use `an` before `1.0`, not `a`             retext-indefinite-article  retext-indefinite-article
```

**Comment by tesseralis** [comment]
This seems to be a common issue and it seems impractical to try to include any unit that begins with a "one" (100m, 1s, 1rem, etc.). Maybe we can adjust the [parser](https://github.com/retextjs/retext-indefinite-article/blob/master/lib/index.js#L122) so that if a number leads a word it's treated as standalone?

**Comment by wooorm** [comment]
Hi Nat! Welcome 👋

Treating them as separate words was a great idea, released in [`2.0.1`](https://github.com/retextjs/retext-indefinite-article/releases/tag/2.0.1)!
